Source/PlasticSourceControl/Private/PlasticSourceControlUtils.cpp

```diff
@@ -1129,31 +1146,61 @@ bool RunGetShelve(const int32 InShelveId, FString& OutComment, FDateTime& OutDat
 
 #endif
 
+bool RunGetChangesets(const FDateTime& InFromDate, TArray<FPlasticSourceControlChangesetRef>& OutChangesets, TArray<FString>& OutErrorMessages)
+{
+	bool bCommandSuccessful = false;
+
+	const FScopedTempFile ChangesetResultFile;
+	TArray<FString> Results;
+	TArray<FString> Errors;
+	TArray<FString> Parameters;
+	Parameters.Add(TEXT("changesets"));
+	if (InFromDate != FDateTime())
+	{
+		Parameters.Add(FString::Printf(TEXT("\"where date >= '%d/%d/%d'\""),
+			InFromDate.GetYear(), InFromDate.GetMonth(), InFromDate.GetDay(),
+			InFromDate.GetYear(), InFromDate.GetMonth(), InFromDate.GetDay()
+		));
```

Comment:
I think this is a copy/past oversight, probably from

```
Parameters.Add(FString::Printf(TEXT("\"where date >= '%d/%d/%d'\" or changesets >= '%d/%d/%d'"),
    InFromDate.GetYear(), InFromDate.GetMonth(), InFromDate.GetDay(),
    InFromDate.GetYear(), InFromDate.GetMonth(), InFromDate.GetDay()
));
```

where there are actually 6 parameters.